### PR TITLE
fixed tooltip

### LINKED ISSUE #4446: Rename the "Add a note to the map button" to "Report an error on map"
### Problem

The current title does not capture the essence and encourages misuse

### Description

Снимок экрана 2023-12-30 в 12 36 00

1. The name does not imply that you can use it to report errors
2. The current name causes people to leave personal notes on the map.

What similar buttons look like on other mapping sites:

<details>
<summary>Yandex Maps</summary>

Снимок экрана 2023-12-30 в 12 46 58

</details>

<details>
<summary>Mapbox</summary>

Снимок экрана 2023-12-30 в 12 47 56

</details>

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by SharanRP**
closes #4446

### CODE REVIEW

**Comment by SharanRP** [comment]
also closes #4446

File: config/locales/en-GB.yml [hunk 1]
```diff
@@ -1367,6 +1367,7 @@ en-GB:
       image size hint: (square images at least 100x100 work best)
       home location: Home Location
       no home location: You have not entered your home location.
+      home location warning : This location will be revealed to other users.
```
Comment (AntonKhorev) [inline config/locales/en-GB.yml:1370]: You only need to edit config/locales/en.yml.

**Comment by AntonKhorev** [comment]
You wanted to say that it also closes #4507. But why didn't you make two pull requests instead? Those are different issues that don't depend on each other.

File: config/locales/en.yml [hunk 2]
```diff
@@ -3095,7 +3095,7 @@ en:
     site:
       edit_tooltip: Edit the map
       edit_disabled_tooltip: Zoom in to edit the map
-      createnote_tooltip: Add a note to the map
+      createnote_tooltip: Move the marker and write a note to fix map errors.
```
Comment (AntonKhorev) [inline config/locales/en.yml:3098]: You are writing about the marker in a tooltip that is shown before the marker is visible.

**Comment by tomhughes** [comment]
As @AntonKhorev says these should be two separate PRs so please remove the second commit and submit it separately.

File: config/locales/en.yml (hunk 2 above)
Comment (SharanRP) [reply config/locales/en.yml:3098]: what should be written then?

**Review by SharanRP — COMMENTED** [review]
I have made the required changes

File: config/locales/en-GB.yml [hunk 3]
```diff
@@ -2098,7 +2098,7 @@ en-GB:
     site:
       edit_tooltip: Edit the map
       edit_disabled_tooltip: Zoom in to edit the map
-      createnote_tooltip: Add a note to the map
+      createnote_tooltip: Add a note to the map.
```
Comment (tomhughes) [inline config/locales/en-GB.yml:2101]: Please don't make changes to `en-GB.yml` or any other locale file other than `en.yml` as translations need to be done on Translatewiki not here.

File: config/locales/en.yml [hunk 4]
```diff
@@ -3095,7 +3096,7 @@ en:
     site:
       edit_tooltip: Edit the map
       edit_disabled_tooltip: Zoom in to edit the map
-      createnote_tooltip: Add a note to the map
+      createnote_tooltip: Add a note to the map.
```
Comment (tomhughes) [inline config/locales/en.yml:3099]: This was correct as it was - none of the other tooltips have a full stop.

File: app/views/profiles/edit.html.erb [hunk 5]
```diff
@@ -43,6 +43,9 @@
   <fieldset>
     <legend><%= t ".home location" -%></legend>
     <p id="home_message" class="text-muted m-0<% if current_user.home_location? %> invisible<% end %>"><%= t ".no home location" %></p>
+    <p class="text-muted m-0<% if current_user.home_location? %> invisible<% end %>">
+       <b><%= t(".home_location_warning") %></b>
```
Comment (tomhughes) [inline app/views/profiles/edit.html.erb:47]: This change will no longer work as you've removed the `home_location_warning` string from the locale file - you were asked to move this to a separate PR which would have required moving both this and the string but instead you just removed the string from this PR and left this.

**Review by tomhughes — CHANGES_REQUESTED** [review]

**Comment by SharanRP** [comment]
ok I will open a new PR
